gr-uhd/apps/uhd_fft

```diff
@@ -54,7 +54,7 @@ except ImportError:
     from gnuradio.uhd.uhd_app import UHDApp
 
 
-class uhd_fft(gr.top_block, Qt.QWidget, UHDApp):
+class uhd_fft(UHDApp, gr.top_block, Qt.QWidget):
```

Comment:
Why the resort?

Reply:
Something something Python3k. This defines an inheritance order (uhd_fft -> UHDApp -> top_block -> Widget), which means that UHDApp "inherits" from Widget. If it's the other way, then invoking Widget will call the ctor on UHDApp, which fails. The other way, it works.